packages/generator/templates/app/app/auth/components/LoginForm.tsx

```diff
@@ -36,6 +36,9 @@ export const LoginForm = (props: LoginFormProps) => {
       >
         <LabeledTextField name="email" label="Email" placeholder="Email" />
         <LabeledTextField name="password" label="Password" placeholder="Password" type="password" />
+        <div>
+          <Link href="/forgot-password">Forgot your password?</Link>
```

Comment:
Should this be?

```suggestion
          <Link href="/forgot-password"><a>Forgot your password?</a></Link>
```

Reply:
You only need the a tag if you want to add classes to it. But probably makes sense to add it.